module/raid/raid.py

```diff
@@ -246,6 +249,13 @@ def raid_execute_once(self, mode, raid):
             Campaign_UseAutoSearch=False,
             Fleet_FleetOrder='fleet1_all_fleet2_standby'
         )
+
+        if mode == 'ex':
+            self.config.override(
+                Submarine_Fleet=1,
+                Submarine_Mode='every_combat'
+            )
```

Comment:
1) `self.config.override` will last until task end, may cause unexpected behaviour.
2) `Submarine_Fleet` and `Submarine_Mode` can still be set to const value without adding argument group `Submarine` to task `Raid`

Reply:
1. That's right usually should have put in a sort of before and after restore back to its original value I thought `override` did this. Is this alright?
```
backup = self.config.Submarine_Fleet
self.config.Submarine_Fleet = 1
...
self.config.Submarine_Fleet = backup
```
2. Adding to the argument group was for users that needed submarine to be configurable since `Raid`. I will remove the argument group since I guess in general you'd want to avoid using submarine in other difficulties besides EX mode.

Reply:
1.
```python
backup = self.config.temporary(key=value)
...
backup.recover()
```
2. Normal battles have submarine hunt and calls, but raid can dont have submarine hunt. If users have their submarine fleet prepared in raid, it's prefered to call them in combat, otherwise users just clear out the submarine fleet. So I think it unnecessary to have a standalone config.